modules/terraform/show.go

```diff
@@ -0,0 +1,26 @@
+package terraform
+
+import (
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// Show runs terraform show with the given options and returns stdout/stderr.
```

Comment:
NIT: Mention it runs `show` with `-json`, so the output will be in JSON format. BTW, should this _only_ return `stdout` so you get solely the JSON output, and none of the logging around it?

Reply:
Oh, I just realized this description doesn't match the one for `Show` below...

Reply:
Updated to sync the two descriptions and also made it so that `Show` only returns stdout: 7a3339e6ae1b0a031913a09312c420f35bce923e